tdrs-backend/tdpservice/users/api/login.py

```diff
@@ -207,3 +250,106 @@ def get(self, request, *args, **kwargs):
                 },
                 status=status.HTTP_400_BAD_REQUEST,
             )
+
+
+class TokenAuthorizationLoginDotGov(TokenAuthorizationOIDC):
```

Comment:
Each `TokenAuthorizationOIDC` subclass will implement `decode_payload`, `get_token_endpoint_response`, and `get_auth_options`